main.go

```diff
@@ -0,0 +1,187 @@
+package main
+
+import (
+	"bufio"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"flag"
+	log "github.com/Sirupsen/logrus"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/zlint"
+	"os"
+	"runtime"
+	"sync"
+	"github.com/zmap/zlint/lints"
+)
+
+var ( //flags
+	inPath           string
+	outPath          string
+	numCertThreads   int
+	prettyPrint      bool
+	numProcs         int
+	channelSize      int
+	crashIfParseFail bool
+	format           string
+)
+
+func init() {
+	flag.StringVar(&inPath, "input-file", "", "File path for the input certificate(s).")
+	flag.StringVar(&outPath, "output-file", "-", "File path for the output JSON.")
+	flag.BoolVar(&prettyPrint, "list-lints-json", false, "Use this flag to print supported lints in JSON format, one per line")
```

Comment:
[minor] I would write these docs like, "Prints supported lints in JSON format, …"

The "Use this flag…" language is unnecessary.